bevy_replicon_renet/src/lib.rs

```diff
@@ -0,0 +1,288 @@
+
+To connect to the server or create it, you need to initialize the
+[`RenetClient`] and [`NetcodeClientTransport`] **or**
+[`RenetServer`] and [`NetcodeServerTransport`](renet::transport::NetcodeServerTransport) resources from Renet.
+
+Never insert client and server resources in the same app for single-player, it will cause a replication loop.
+
+This crate provides [`RenetChannelsExt`] extension trait to conveniently convert channels
+from [`RepliconChannels`] resource into renet channels.
+So when creating server or client you need to initialize [`ConnectionConfig`](renet::ConnectionConfig)
+from [`renet`] like this:
+
+```
+use bevy::prelude::*;
+use bevy_replicon::prelude::*;
+use bevy_replicon_renet::{renet::ConnectionConfig, RenetChannelsExt, RepliconRenetPlugins};
+
+# let mut app = App::new();
+# app.add_plugins(RepliconPlugins);
+let channels = app.world.resource::<RepliconChannels>();
+let connection_config = ConnectionConfig {
+    server_channels_config: channels.get_server_configs(),
+    client_channels_config: channels.get_client_configs(),
+    ..Default::default()
+};
+```
+
+For a full example of how to initialize server or client see the example in the
+repository.
+*/
+
+use bevy::{app::PluginGroupBuilder, prelude::*};
+pub use bevy_renet::*;
+use bevy_replicon::prelude::*;
+use renet::{ChannelConfig, ClientId, RenetClient, RenetServer, SendType, ServerEvent};
+#[cfg(feature = "renet_transport")]
+use {
+    renet::transport::NetcodeClientTransport,
+    transport::{NetcodeClientPlugin, NetcodeServerPlugin},
+};
+
+pub struct RepliconRenetServerPlugin;
+
+impl Plugin for RepliconRenetServerPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_plugins(RenetServerPlugin)
+            .add_systems(
+                PreUpdate,
+                (
+                    Self::peer_events_system.in_set(ServerSet::PeerEvents),
+                    (
+                        Self::starting_system.run_if(resource_added::<RenetServer>),
+                        Self::stopping_system.run_if(resource_removed::<RenetServer>()),
+                        Self::receiving_packets.run_if(resource_exists::<RenetServer>),
+                    )
+                        .in_set(ServerSet::ReceivePackets),
+                )
+                    .after(RenetReceive),
+            )
+            .add_systems(
+                PostUpdate,
+                Self::sending_system
+                    .before(RenetSend)
+                    .in_set(ServerSet::SendPackets)
+                    .run_if(resource_exists::<RenetServer>),
+            );
+
+        #[cfg(feature = "renet_transport")]
+        app.add_plugins(NetcodeServerPlugin);
+    }
+}
+
+impl RepliconRenetServerPlugin {
+    fn starting_system(mut server: ResMut<RepliconServer>) {
+        server.set_running(true);
+    }
+
+    fn stopping_system(mut server: ResMut<RepliconServer>) {
+        server.set_running(false);
+    }
+
+    fn peer_events_system(
```

Comment:
It's a good point.

I used `_system` everywhere in the codebase because `@cart` prefers this naming convention. I remember [this](https://github.com/bevyengine/bevy/discussions/2804) discussion and it looks like the community is in favor of not having the `_system` prefix. But this was back in the day when you had to call `.system()` on every system like this: `damage_system.system()` and it was annoying. When `.system()` was removed, the discussions stopped.

But I did a quick search in discord and [according to `@alice_i_cecile`](https://discord.com/channels/691052431525675048/691052431974465548/1131707341256020078) the decision was made to not use `_system`.

So maybe keep it as is in this PR and right after it I will submit a follow-up to rename the whole codebase at once?